### DISCUSSION TITLE
A way to make xdotool key work with Foliate?

### DISCUSSION THREAD

**Discussion by zheneknew** [johnfactotum/foliate#1188] [General]
**Question:**
Is there a way to build the 2.6.4 version in such a way that it is able to retrieve keys from xdotool when its window is not active(because e.g. it's on a second monitor)?

This doesn't work:
```
xdotool search --onlyvisible --classname "Foliate" key --clearmodifiers j
```

This works (if press Enter and make the Foliate window active fastly):
```
sleep 3; xdotool search --onlyvisible --classname "Foliate" key --clearmodifiers j
```

**Version:**
 - Foliate version: 2.6.4
 - OS/Distribution and version: Fedora 38
 - Desktop environment: Gnome 43
 - Installation method: dnf

**Comment by johnfactotum** [comment]
I don't know about xdotool but some shortcuts only works if the webview is focused. This includes h/j/k/l, arrow keys, page up/down. N and P, on the other hand, are global shortcuts.
